packages/flame_markdown/lib/custom_attribute_syntax.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:markdown/markdown.dart';
+
+// cSpell:ignore charcode.dart (file name from another package)
+// NOTE: values obtained from file `charcode.dart` from the markdown package
+class _Chars {
```

Comment:
Does this even need a class?

Reply:
No but I thought it was more organized to group these together than let them laying around